wpimath/src/main/java/edu/wpi/first/math/estimator/KalmanFilter.java

```diff
@@ -37,12 +37,22 @@
 
   /** The steady-state Kalman gain matrix. */
   @SuppressWarnings("MemberName")
-  private final Matrix<States, Outputs> m_K;
+  private Matrix<States, Outputs> m_K;
 
   /** The state estimate. */
   @SuppressWarnings("MemberName")
   private Matrix<States, N1> m_xHat;
 
+  @SuppressWarnings("MemberName")
+  private Matrix<States, States> m_P;
+
+  private Matrix<States, States> m_contQ;
+  private Matrix<Outputs, Outputs> m_contR;
+
+  private final Matrix<States, States> m_discA;
+  private final Matrix<States, States> m_discQ;
```

Comment:
m_discA and m_discQ aren't needed; the local variables discA and discQ are sufficient. C++ has them to avoid allocating more storage, but Java allocates new objects in discretizeAQ() anyway.

Reply:
Addressed in commit ad9900c